### ISSUE TITLE
Use rust-bitcoin feature instead of base64 dep

### ISSUE THREAD

**Issue by RCasatta** [Kixunil/loptos#1]
https://github.com/Kixunil/loptos/blob/965aeda80a0e81b3250585020608e2e9ab533542/Cargo.toml#L17

rust-bitcoin has feature base64 adding the capability to create a PSBT from a base64 string and viceversa

**Comment by Kixunil** [comment]
Ah, `bitcoin` docs sucks when it comes to features. Will look into it.

**Comment by Kixunil** [comment]
Hmm, I'm not sure this is really that great idea. It saves a three lines of code but `base64` is still transitive dependency of multiple crates, so now I have two base64 implementations.

**Comment by RCasatta** [comment]
> Hmm, I'm not sure this is really that great idea. It saves a three lines of code but `base64` is still transitive dependency of multiple crates, so now I have two base64 implementations.

right, I see other deps depend on base64 v0.13.0 so yes better to not use base64 feature so it's used only one base64 implementation and wait for https://github.com/rust-bitcoin/rust-bitcoin/issues/510#issuecomment-888903597

**Comment by DanGould** [comment]
Is using rust-bitcoin base64 feature ok now since https://github.com/rust-bitcoin/rust-bitcoin/issues/510#issuecomment-888903597 510 is closed?

**Comment by Kixunil** [comment]
Oh, yeah, thanks for heads up!
